assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -142,13 +147,21 @@ cmdline_realtime=+nohz_full=${isolated_cores} tsc=nowatchdog nosoftlockup nmi_wa
 {{end}}
 
 {{if .HighPowerConsumption}}
-cmdline_power_performance=+processor.max_cstate=1 intel_idle.max_cstate=0 intel_pstate=disable
+cmdline_power_performance=+processor.max_cstate=1 intel_idle.max_cstate=0 
 {{end}}
 
 {{if and .HighPowerConsumption .RealTime}}
 cmdline_idle_poll=+idle=poll
 {{end}}
 
+{{if .PerPodPowerManagement}}
+cmdline_pstate=+intel_pstate=passive
```

Comment:
Do we want to set this explicitly or just rely on default? Or is some other part of the tuned profile inheriting something we must override? Please add a comment explaining the rationale here.

Reply:
I would prefer this is explicit. I believe the cpu-partitioning profile is setting this to intel_pstate=disable so we need to override it.

Reply:
Right, there is and if else for that:

`{{if .PerPodPowerManagement}}
cmdline_pstate=+intel_pstate=passive
{{end}}

{{if and (not .PerPodPowerManagement) .HighPowerConsumption}}
cmdline_pstate=+intel_pstate=disable
{{end}}`